state/executor.go

```diff
@@ -210,14 +211,22 @@ func (e *BlockExecutor) updateState(state types.State, block *types.Block, abciR
 		if len(validatorUpdates) > 0 {
 			err := nValSet.UpdateWithChangeSet(validatorUpdates)
 			if err != nil {
-				return state, nil
+				if errors.Is(err, ErrEmptyValSetGenerated) {
```

Comment:
this should be `is not` instead of `is`? because we want to accept the `ErrEmptyValSetGenerated` case?

Reply:
great catch, thanks! fixed

Reply:
seems like using `errors.Is` was wrong because `errors.New` creates a new error object and `Is` compares if it's the object.